File: back/src/main/java/com/tdevilleduc/urthehero/back/dao/PageDao.java
package com.tdevilleduc.urthehero.back.dao;

import com.tdevilleduc.urthehero.back.model.Page;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface PageDao extends JpaRepository<Page, Integer> {

}

File: back/src/main/java/com/tdevilleduc/urthehero/back/dao/PageDao.kt
package com.tdevilleduc.urthehero.back.dao

import com.tdevilleduc.urthehero.back.model.Page
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface PageDao : JpaRepository<Page, Int> {
    fun countByStoryId(storyId: Int): Long
}